content/updates/2020-06-15-update.md

```diff
@@ -1,10 +1,10 @@
 +++
-title = "~2020.6.15 urbit.org update"
+title = "~2020.6.15 Update"
 date = 2020-06-15
 description = "A brief update from Tlon for ~2020.6.15"
 slug = "2020-06-15"
 [extra]
-author = "Corrina"
+author = 
```

Comment:
```suggestion
author = ""
```

Reply:
Putting nothing in here crashes the site build — empty string it.

Reply:
ah thanks